client/my-sites/domains/domain-management/list/item-placeholder.jsx

```diff
@@ -8,7 +8,7 @@ import React from 'react';
  */
 import CompactCard from 'components/card/compact';
 
-const ListItemPlaceholder = React.createClass( {
+class ListItemPlaceholder extends React.PureComponent {
 	render() {
 		return (
 			<CompactCard className="domain-management-list-item is-placeholder">
```

Comment:
Yep, but these are a PITA to fix as we need to move all the CSS. That's why I skip those. :)